charts/prometheus-blackbox-exporter/templates/deployment.yaml

```diff
@@ -86,6 +86,10 @@ spec:
               {{- if .Values.runAsGroup }}
             runAsGroup: {{ .Values.runAsGroup }}
               {{- end }}
+              {{- if .Values.dropAllCapabilities }}
+            capabilities:
```

Comment:
is defined twice: https://github.com/prometheus-community/helm-charts/blob/65b9e4fcf1ffdeac6cc3317d49cc7874bad591f9/charts/prometheus-blackbox-exporter/templates/deployment.yaml#L79

Reply:
I did some refactor to simplify the syntax and avoid duplication